process/OpenHW-Specification-Process.md

```diff
@@ -0,0 +1,180 @@
+- "OpenHW Group Specification - In Review" (optional)
+- "OpenHW Group Specification - Release Candidate"
+- "OpenHW Group Specification - Released"
+
+#### "OpenHW Group Specification - In Development"
+
+When the specification is initiated, all drafts will be entitled  "OpenHW Group Specification: Title : VX.Y.Z : In Development"
+(During initial development of the spec before first release, the Revision number is of the form X.Y.Z with X=0.) 
+
+This state is also used when a released spec is subsequently revised. 
+
+During revision of a released specification, the Revision number increments as appropriate depending on whether the revision is a major or minor update. 
+
+#### "OpenHW Group Specification - In Review"
+
+This state is optional; see the note below.
+
+When the specification is frozen for technical review by a Task Group, all drafts will be entitled "OpenHW Group Specification: Title : VX.Y.Z : In Review"
+When the TG(s) has completed its review, the reviewed and stable Revision will be relabelled as "Release Candidate" - see next section.
+
+NOTE: This state can be bypassed if a Task Group does not require a review stage. That is, the TG can move the spec from "In Development" to "Release Candidate".
+
+
+#### "OpenHW Group Specification - Release Candidate"
+
+When the specification is proposed as a candidate for release, drafts will be entitled "OpenHW Group Specification: Title : VX.Y.Z  : Release Candidate"
+
+The Revision number used at this stage would normally be same as the proposed Revision to be released. 
+
+The Release Candidate specification will be notified as open for review to the OpenHW TWG.  
+Although technical comments are expected to have been resolved by the TG, comments may still be accepted at this stage.
+After a suitable review period, the TWG Chair/Co-Chair will determine that the Specification is ready for ballot. 
+
+
+#### "OpenHW Group Specification - Released"
+
+When a Release Candidate specification has completed ratification (through a TWG vote), the released copy will be entitled "OpenHW Group Specification: Title : VX.Y.Z  : Released"
+Revision numbers should be of the form X.Y.Z, normally starting with X=1 as the initially released version.
+
+The Github commit corresponding to the Released spec will be listed on the Github page of the specification repo together with the "OpenHW Group Specification: Title : VX.Y.Z : Released" string
+
+
+## Specification Format
+
+The following sections must be included in the OpenHW Group Specification
+
+### Title
+
+The Title should be of the form "OpenHW Group Specification: Title : VX.Y.Z : State"
+
+### Revision History
+
+Prior to release a table of draft revisions should be included, which can include description of content in each draft
+
+Upon release, the Revision History should list only released specification Revisions. That is, intermediate revisions used during development and review don't need to be listed.
```

Comment:
The revision history is generated from the releases on github. There is no control over which versions should be listed.